### ISSUE TITLE
Odd example for `alt` attribute in "4.7.5.1.8. A graphical representation of some of the surrounding text"

### ISSUE THREAD

**Issue by aaaxx** [w3c/html#958]
The section 4.7.5.1.8, "A graphical representation of some of the surrounding text", says that the effectively superfluous images that just reinforce the surrounding text should have a blank `alt` attribute (presumably so that screen readers can just skip them), but the example given contradicts that. (It actually kind of looks like an earlier draft of the example 39 in the previous section, 4.7.5.1.7, "Images that enhance the themes or subject matter of the page content".)
